sympy/physics/vector/frame.py

```diff
@@ -1502,6 +1503,16 @@ def partial_velocity(self, frame, *gen_speeds):
         else:
             return tuple(partials)
 
+    def _latex(self, printer):
+        return r"{%s}" % self.name
+
+    def _pretty(self, printer):
+        e = self
+        class Fake:
+            def render(self, *args, **kwargs):
+                return r"{%s}" % e.name
+        return Fake()
```

Comment:
What does `Fake` mean?

Reply:
I'm following what was implemented for physics.vector. But yes, from what I gathered the printer pipeline needs for the _pretty method in a printable class to return an object with a method called "render" implemented.